modules/web/client/src/main/scala/observe/ui/components/MainApp.scala

```diff
@@ -136,14 +139,15 @@ object MainApp extends ServerEventHandler:
 
   // Log in from cookie and switch to staff role
   private def enforceStaffRole(ssoClient: SSOClient[IO]): IO[Option[UserVault]] =
-    ssoClient.whoami.flatMap: userVault =>
-      userVault.map(_.user) match
-        case Some(StandardUser(_, role, other, _)) =>
-          (role +: other)
-            .collectFirst { case StandardRole.Staff(roleId) => roleId }
-            .fold(IO(userVault))(ssoClient.switchRole)
-        // .map(_.orElse(throw new Exception("User is not staff")))
-        case _                                     => IO(userVault)
+    ssoClient.whoami
+      .flatMap: userVault =>
+        userVault.map(_.user) match
+          case Some(StandardUser(_, role, other, _)) =>
+            (role +: other)
+              .collectFirst { case StandardRole.Staff(roleId) => roleId }
+              .fold(IO(userVault))(ssoClient.switchRole)
+          // .map(_.orElse(throw new Exception("User is not staff")))
```

Comment:
Commented code

Reply:
Thank you. This one is on purpose. We may have to find a better mechanism to handle this. Will the STAFF role be enough to operate oberve?